sql/sql_table.cc

```diff
@@ -10155,9 +10167,20 @@ bool mysql_alter_table(THD *thd, const LEX_CSTRING *new_db,
   if (WSREP(thd) &&
       (thd->lex->sql_command == SQLCOM_ALTER_TABLE ||
        thd->lex->sql_command == SQLCOM_CREATE_INDEX ||
-       thd->lex->sql_command == SQLCOM_DROP_INDEX) &&
-      !wsrep_should_replicate_ddl(thd, table_list->table->s->db_type()->db_type))
-    DBUG_RETURN(true);
+       thd->lex->sql_command == SQLCOM_DROP_INDEX))
+  {
+    /*
+       For partitioned tables we need to resolve implenting
```

Comment:
Typo: implenting -> implementing
And a suggestion: better word would be "underlying" storage engine?

Reply:
Fixed